### PR TITLE
added readme file to be shown on the github repository frontpage

### PR DESCRIPTION
**Pull request by Dzejrou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Note: If you installed the toolchain to a custom directory, make sure `CROSS_PREFIX`
+environment variable is correctly set.
+
+### Running the OS
+
+When you get the command line back, there should be an `image.iso` file in the source
+root directory. If you have QEMU, you should be able to start HelenOS by running:
+
+```
+$ qemu-system-i386 -m <t_co> -boot d -cdrom image.iso
```
Comment (le-jzr) [inline README.md]: Should be `-x86_64`?

Comment (jermar) [reply README.md]: I'd say that under all circumstances, the user should run tools/ew.py.

Comment (le-jzr) [reply README.md]: Yeah.

Comment (Dzejrou) [reply README.md]: Yes, I missed that. Also, the <t_co> after -m wierds me out, that's supposed to be 256. Now that I went to fix it in the file, it seems that anytime I write 256 in vim it gets changed to 256 :D

Comment (le-jzr) [reply README.md]: > anytime I write 256 in vim it gets changed to 256
Not entirely surprising. :D

Comment (Dzejrou) [reply README.md]: ... Meant <t_co> :)

Comment (Dzejrou) [reply README.md]: Fixed.

**Pull request merged**